### PR TITLE
add armadillo linear algebra library

### PR DESCRIPTION
**Pull request by golgepapaz**

### CODE REVIEW

File: mingw-w64-armadillo/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+_realname=armadillo
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=5.100.2
+pkgrel=1
+pkgdesc="C++ linear algebra library (mingw-w64)"
+arch=('any')
+url="http://arma.sourceforge.net/"
+license=(MPL2)
+depends=("${MINGW_PACKAGE_PREFIX}-openblas"
+		 "${MINGW_PACKAGE_PREFIX}-lapack")
```
Comment (Alexpux) [inline mingw-w64-armadillo/PKGBUILD]: Don't use TAB's. Use spaces instead

File: mingw-w64-armadillo/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  cd "$srcdir/${_realname}-${pkgver}"
+  mkdir -p build-${MINGW_CHOST}
+  cd build-${MINGW_CHOST}
+  
+  #  PATH=${MINGW_PREFIX}/bin \
+  ${MINGW_PREFIX}/bin/cmake.exe \
+    -G"MSYS Makefiles" \
+    -DCMAKE_INSTALL_PREFIX=${pkgdir}${MINGW_PREFIX} \
+    -DCMAKE_BUILD_TYPE=Release \
+	..
```
Comment (Alexpux) [inline mingw-w64-armadillo/PKGBUILD]: Here TAB too

File: mingw-w64-armadillo/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+}
+package() {
+cd "${srcdir}/${_realname}-${pkgver}/build-${MINGW_CHOST}"
+make install
```
Comment (Alexpux) [inline mingw-w64-armadillo/PKGBUILD]: Add two spaces at the start of the lines inside procedure

**Comment by golgepapaz** [comment]
fix whitespace

**Pull request merged**